### DISCUSSION TITLE
Validation at form level

### DISCUSSION THREAD

**Discussion by frerotg** [vueform/vueform#327] [Questions]
Hello I would like to execute a validation async request on form level. Currently **Validators** can be created at element level, but in my case there is inter-element validation. Depending on changes to an element, some elements may become invalid or other. 

To simplify things, I have an async request that sends the entire form and receives the status of each element (error or not, with a dedicated error message). 

I can't find any way of doing this. I've managed to define custom error messages for certain elements but the ‘invalid’ option for these remains false even though they have an error message.

My idea was to have a validate event at the form level and this request would return the list of validations for each element of the form.

Best Regards

**Comment by adamberecz** [comment]
If a form field (e.g., fieldA) is flagged as invalid by the backend, it prevents the form from being submitted. Once the field is set to an invalid state, how can it be updated to a valid state if validation can only occur after form submission, which is disabled while the field remains invalid?
